types/src/transaction/mod.rs

```diff
@@ -738,6 +739,49 @@ impl TransactionWithProof {
     }
 }
 
+/// The status of VM execution, which contains more detailed failure info
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+#[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
+#[cfg_attr(any(test, feature = "fuzzing"), proptest(no_params))]
+pub enum TransExecutionStatus {
+    Executed,
```

Comment:
Instead of `Executed`, should this be renamed to `Success` for clarity?

Reply:
I like this name so much and I am gonna change all the callsites from executed to Success.